packages/sdk/src/provider/extensionProviderHelpers/handleBatchMethod.ts

```diff
@@ -29,10 +33,33 @@ export const handleBatchMethod = async ({
   }
 
   const resp = await target.request(args);
+
+  const { dappMetadata } = sdkInstance;
+  const url = dappMetadata?.url ?? 'no_url';
+  const name = dappMetadata?.name ?? 'no_name';
+  const id = getOrCreateUuidForIdentifier(url, name);
+
+  const platFormType = sdkInstance.platformManager?.getPlatformType();
```

Comment:
isn't that a lot cleaner solution @andreahaku 

If we have already a method that can directly tell us if this is the `extension` that is connected or not, that seems to be the way to solve this.

One of the things that this would not attribute correctly using this method is if you are on "desktop" browser but you connect to your mobile "wallet".

In that case the "from" should be "mobile" and platform should be "Desktop-web"
while with your solution the "from" would be "extension" and platform would be "Desktop-web"